composer.json

```
@@ -23,7 +23,7 @@
         "silverstripe/framework": "^4.10",
         "silverstripe/reports": "^4.4",
         "symbiote/silverstripe-queuedjobs": "^4",
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^6.3 || ^7"
```

```suggestion
        "guzzlehttp/guzzle": "^6.3 || ^7.0"
```
Just make it a little more consistent

---

I have made this change.